doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -119,6 +119,11 @@ To configure the LIT, CIP, UAT or DSLS, use the following Kconfig options:
   The DSLS support allows the application to dynamically switch between SIT and LIT modes, as long as the requirements for these modes are met.
   This option is by default disabled for the LIT device.
 
+You can enable optional reporting on entering the active mode.
+When enabled, the device sends a data report to the subscribed devices.
+This could be useful especially in the combination with the User Active Mode Trigger (UAT) feature, to inform the subscribed Matter controller that the user triggered an ICD to enter the active mode.
```

Comment:
Shouldn't it be (UAMT)?

Reply:
Yeah, you are right, but this is the shortcut and explanation straight from the Matter spec, so I believe we should keep it as is.